--- cmake/config.h.in.orig	2021-03-24 18:50:33.000000000 -0700
+++ cmake/config.h.in	2021-03-24 18:51:10.000000000 -0700
@@ -28,7 +28,7 @@
 #cmakedefine _DARWIN_C_SOURCE
 
 // Define to activate features from IEEE Stds 1003.1-2008.
-#define _POSIX_C_SOURCE 200809L
+//#define _POSIX_C_SOURCE 200809L
 
 #if defined(__SunOS_5_8) || defined(__SunOS_5_9) || defined(__SunOS_5_10)
 #  define _XOPEN_SOURCE 500
